using-mobile-extensions/adobe-media-analytics/migration.md

```diff
@@ -0,0 +1,947 @@
+  @objc(AEPMediaStreamType)
+  public class StreamType: NSObject {
+     // Constant defining stream type for VOD streams.
+        public static let VOD = "vod"
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x \(Objective-C\)" %}
+
+```objective-c
+/**
+ * Constant defining stream type for VOD streams
+ */
+FOUNDATION_EXPORT NSString* _Nonnull const ACPMediaStreamTypeVod;
+
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+{% hint style="info" %}
+For the full list of constant type, refer to [Media API reference]((media-api-reference#stream-type)).
+{% endhint %}
+
+### Standard video constants
+
+{% tabs %}
+
+{% tab title="AEP 3.x \(Swift\)" %}
+
+```swift
+public class VideoMetadataKeys: NSObject {
+        public static let SHOW = "a.media.show"
+}
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x \(Objective-C\)" %}
+
+```objective-c
+public class MediaConstants: NSObject {
+  @objc(AEPVideoMetadataKeys)
+  public class VideoMetadataKeys: NSObject {
+        public static let SHOW = "a.media.show"
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x \(Objective-C\)" %}
+
+```objective-c
+FOUNDATION_EXPORT NSString* _Nonnull const ACPVideoMetadataKeyShow;
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+{% hint style="info" %}
+For the full list of constant type, refer to [Media API reference](media-api-reference#standard-video-constants).
+{% endhint %}
+
+### Standard audio constants
+
+{% tabs %}
+
+{% tab title="AEP 3.x \(Swift\)" %}
+
+```swift
+public class AudioMetadataKeys: NSObject {
+        public static let ARTIST = "a.media.artist"
+}
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x \(Objective-C\)" %}
+
+```objective-c
+public class MediaConstants: NSObject {
+  @objc(AEPAudioMetadataKeys)
+  public class AudioMetadataKeys: NSObject {
+        public static let ARTIST = "a.media.artist"
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x \(Objective-C\)" %}
+
+```objective-c
+FOUNDATION_EXPORT NSString* _Nonnull const ACPAudioMetadataKeyArtist;
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+{% hint style="info" %}
+For the full list of constant type, refer to [Media API reference](media-api-reference#standard-audio-constants).
+{% endhint %}
+
+### Standard ad constants
+
+{% tabs %}
+
+{% tab title="AEP 3.x \(Swift\)" %}
+
+```swift
+ public class AdMetadataKeys: NSObject {
+        public static let ADVERTISER = "a.media.ad.advertiser"
+ }
+```
+
+{% endtab %}
+
+{% tab title="AEP 3.x \(Objective-C\)" %}
+
+```objective-c
+public class MediaConstants: NSObject {
+  @objc(AEPAdMetadataKeys)
+  public class AdMetadataKeys: NSObject {
+        public static let ADVERTISER = "a.media.ad.advertiser"
+ }
+}
+```
+
+{% endtab %}
+
+{% tab title="ACP 2.x \(Objective-C\)" %}
+
+```objective-c
+FOUNDATION_EXPORT NSString* _Nonnull const ACPAdMetadataKeyAdvertiser;
+```
+
+{% endtab %}
+
+{% endtabs %}
+
+{% hint style="info" %}
+For the full list of constant type, refer to [Media API reference](media-api-reference#standard-ad-constants).
```

Comment:
Does the link like this work properly?

Reply:
It should work. I will also test once it is merged.